integration/tests/epochs/recover_epoch/recover_epoch_efm_test.go

```diff
@@ -19,16 +23,79 @@ type RecoverEpochSuite struct {
 	Suite
 }
 
-// TestRecoverEpoch ensures that the recover_epoch transaction flow works as expected. This test will simulate the network going
-// into EFM by taking a consensus node offline before completing the DKG. While in EFM mode the test will execute the efm-recover-tx-args
-// CLI command to generate transaction arguments to submit a recover_epoch transaction, after submitting the transaction the test will
-// ensure the network is healthy.
+// TestRecoverEpoch ensures that the recover epoch governance transaction flow works as expected and a network that
+// enters Epoch Fallback Mode can successfully recover. This test will do the following:
+// 1. Manually triggers EFM by turning off a collection node before the end of the DKG forcing the DKG to fail.
+// 2. Generates epoch recover transaction args using the epoch efm-recover-tx-args.
+// 3. Submit recover epoch transaction.
+// 4. Ensure expected EpochRecover event is emitted.
+// Currently, this test does not test the processing of the EpochRecover event see this issue: https://github.com/onflow/flow-go/issues/6164
 func (s *RecoverEpochSuite) TestRecoverEpoch() {
-	s.AwaitEpochPhase(context.Background(), 0, flow.EpochPhaseSetup, 20*time.Second, time.Second)
-	fmt.Println("in epoch phase setup")
+	// 1. Manually trigger EFM
+	// wait until the epoch setup phase to force network into EFM
+	s.AwaitEpochPhase(s.Ctx, 0, flow.EpochPhaseSetup, 10*time.Second, 500*time.Millisecond)
+	// pausing consensus node will force the network into EFM
+	ln := s.GetContainersByRole(flow.RoleCollection)[0]
+	_ = ln.Pause()
```

Comment:
The comment says consensus node, but we're pausing the collection node. 

In addition, I am guessing that EFM is being triggered only because we have made the DKG phases much shorter (10 views).

We should test this without pausing the collection node:
- if we still trigger EFM, then we don't need to pause the collection node and can remove that code
- if we don't trigger EFM, then we should revert the change to the DKG phase length

Reply:
The test did not work without pausing the collection node, we can look into it further in [part 2](https://github.com/onflow/flow-go/issues/6164).